src/screens/GroupSetting/elements/EditGroup.js

```diff
@@ -107,23 +107,23 @@ const styles = StyleSheet.create({
     marginBottom: 5
   },
   textDesc: {
-    color: COLORS.white,
+    color: COLORS.almostBlack,
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: normalizeFontSize(18)
   },
   container: {
     backgroundColor: COLORS.anon_primary,
     borderTopWidth: 1,
-    borderTopColor: COLORS.lightgrey,
+    borderTopColor: COLORS.gray100,
     paddingVertical: 10,
     paddingHorizontal: 15
   },
   btnUpdatePhoto: {
     width: normalize(48),
     height: normalize(48),
     borderRadius: normalize(24),
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Several changes have been made here, including changing the color of the text in the `textDesc` style from `COLORS.white` to `COLORS.almostBlack`, changing the `borderTopColor` in the `container` style from `COLORS.lightgrey` to `COLORS.gray100`, and changing the `backgroundColor` in the `btnUpdatePhoto` style from `COLORS.lightgrey` to `COLORS.gray100`. These changes could significantly alter the appearance of the UI. Ensure these changes are consistent with the overall design and do not negatively impact usability or accessibility.
```diff
- color: COLORS.white,
+ color: COLORS.almostBlack,

- borderTopColor: COLORS.lightgrey,
+ borderTopColor: COLORS.gray100,

- backgroundColor: COLORS.lightgrey,
+ backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->